R/method-differentialAbundance.R

```diff
@@ -227,6 +232,7 @@ setMethod("maaslin", signature("AbundanceData", "Comparator"), function(data, co
 #' @param data AbsoluteAbundanceData object
 #' @param comparator Comparator object specifying the variable and values or bins to be used in dividing samples into groups.
 #' @param method string defining the the differential abundance method. Accepted values are 'DESeq' and 'ANCOMBC'.
+#' @param pValueFloor numeric value that indicates the smallest p value that should be returned. The default value is P_VALUE_FLOOR defined in this package.
```

Comment:
might make sense to explain the adjusted pvalues are impacted by this param as well.

Reply:
also we should say what the default is..